mnemonic/test/Main.hs

```diff
@@ -0,0 +1,15 @@
+{-# LANGUAGE LambdaCase          #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Main where
+
+import           Universum
+
+import           Test.Hspec
+
+import qualified BIP39Spec
```

Comment:
Let's reflect the library file structure in the specs, and go for: `Cardano.MnemonicSpec`